.dockerignore

```diff
@@ -26,3 +26,4 @@ endo-sha.txt
 # `@@AGORIC_DOCKER_SUBMODULES@@`
 packages/xsnap/moddable
 packages/xsnap/xsnap-native
+!packages/xsnap/xsnap-native/xsnap/build/bin
```

Comment:
I'm not convinced this is correct.  The only way it would work is if the platform within the Docker container is (coincidentally) binary-compatible with the host platform's (outside of Docker) compiler toolchain, and the host built the binary before attempting to run the Docker build.

That bundling of a host-specific binary violates the idea of Docker builds being reproducible.

In practical terms, this means that I won't be able to run `agoric-sdk/a3p-integration` on my MacOS box, because the host platform's compiler toolchain is not binary-compatible with the Linux-based system inside the Docker containers.

Reply:
I’ve removed this line from `.dockerignore`. It might be appropriate to remove the submodule exceptions as well, though they appear to be doing no harm.